### ISSUE TITLE
Was anyone able to train on their own dataset without labels?

### ISSUE THREAD

**Issue by Harsh9524** [wvangansbeke/Unsupervised-Classification#111]
I went through the #8 and #19 and read the discussion about training the model on custom dataset. If anyone has successfully trained on custom dataset can they share the repository?

**Comment by edwardcho** [comment]
Hello Sir,

I am looking forward answers for this issue.

Thanks,
Edward Cho.

**Comment by Mohit-robo** [comment]
Yes, you would need to create a Dataloader for your dataset, In my case, I had 2 classes, and create a Dataloader, that's it.

**Comment by itsMohitShah** [comment]
Hi Mohit, 
Can you share the code for that?

I believe you are talking about making your own .py file in data folder 
(data/custom_data.py)
Along with that, could you also please share your version of common_confi?
Will be a massive help.
